Hi @mithomas,

thanks for your pr. Can you fix the current test failure caused by your change? Afterwards it's good to be merged.

```
$ make check-scripts
grep -rIl '^#![[:blank:]]*/bin/\(bash\|sh\|zsh\)' \
	--exclude-dir=.git --exclude=*.sw? \
	| xargs shellcheck
In core/start-service.sh line 120:
		-wait tcp://$KCCONF_SPOOLER_SMTP_SERVER:25 \
                            ^-- SC2086: Double quote to prevent globbing and word splitting.
make: *** [check-scripts] Error 123
```

--- a/core/start-service.sh
+++ b/core/start-service.sh
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 ADDITIONAL_KOPANO_PACKAGES=${ADDITIONAL_KOPANO_PACKAGES:-""}
+KCCONF_SERVER_MYSQL_SOCKET=${KCCONF_SERVER_MYSQL_SOCKET:-""}
 
 set -eu # unset variables are errors & non-zero return values exit the whole script
 
@@ -39,7 +40,11 @@ fi
 case "$SERVICE_TO_START" in
 server)
 	# determine db connection mode (unix vs. network socket)
-	if [ -n "$KCCONF_SERVER_MYSQL_SOCKET" ]; then DB_CONN="file://$KCCONF_SERVER_MYSQL_SOCKET"; else DB_CONN="tcp://$KCCONF_SERVER_MYSQL_HOST:$KCCONF_SERVER_MYSQL_PORT"; fi
+	if [ -n "$KCCONF_SERVER_MYSQL_SOCKET" ]; then
+		DB_CONN="file://$KCCONF_SERVER_MYSQL_SOCKET"
+	else
+		DB_CONN="tcp://$KCCONF_SERVER_MYSQL_HOST:$KCCONF_SERVER_MYSQL_PORT"
+	fi
 	/kopano/services/kopano-public-store.sh &
 	/kopano/services/kopano-users.sh &
 	dockerize \

--- a/core/start-service.sh
+++ b/core/start-service.sh
@@ -117,7 +117,7 @@ search)
 spooler)
 	dockerize \
 		-wait file://var/run/kopano/server.sock \
-		-wait tcp://$KCCONF_SPOOLER_SMTP_SERVER:25 \
+		-wait tcp://"$KCCONF_SPOOLER_SMTP_SERVER":25 \
 		-timeout 1080s
 	# cleaning up env variables
 	unset "${!KCCONF_@}"

